lib/rubocop/cop/naming/binary_operator_parameter_name.rb

```diff
@@ -18,7 +18,7 @@ class BinaryOperatorParameterName < Cop
               'name its argument `other`.'.freeze
 
         OP_LIKE_METHODS = %i[eql? equal?].freeze
-        BLACKLISTED = %i[+@ -@ [] []= << `].freeze
+        BLACKLISTED = %i[+@ -@ [] []= << === `].freeze
```

Comment:
On a note separate to your changes, `BLACKLISTED` seems like a very inappropriate name for this constant. 🙂